lib/muffin_man.rb

```diff
@@ -14,7 +14,8 @@
 require "muffin_man/listings/v20210801"
 require "muffin_man/fulfillment_inbound/v0"
 require "muffin_man/fulfillment_inbound/v1"
-
+require "muffin_man/feeds/v20210630"
```

Comment:
@shubham-n-khanna I think we need to update README file as well.

Reply:
Yes @ssontakke once PR will be merged I will update README